Typescript: "react" has no exported member named 'ForwardedRef'

## Problem

Currently there's a problem with the exported function `ForwardedRef` on file `scr/types.ts`.

This problem was introduced on #782 (https://github.com/reactchartjs/react-chartjs-2/pull/782/files#diff-c54113cf61ec99691748a3890bfbeb00e10efb3f0a76f03a0fd9ec49072e410aR3).

This causes the Typescript compiler to fail with the following problems:

```bash
> tsc

node_modules/react-chartjs-2/dist/chart.d.ts:4:17 - error TS2724: 'React' has no exported member named 'ForwardedRef'. Did you mean 'forwardRef'?

4     ref?: React.ForwardedRef<import("./types").ChartJSOrUndefined<TType, TData, TLabel>> | undefined;
                  ~~~~~~~~~~~~

node_modules/react-chartjs-2/dist/typedCharts.d.ts:4:17 - error TS2724: 'React' has no exported member named 'ForwardedRef'. Did you mean 'forwardRef'?

4     ref?: React.ForwardedRef<ChartJSOrUndefined<"line", TData, TLabel>> | undefined;
                  ~~~~~~~~~~~~

node_modules/react-chartjs-2/dist/typedCharts.d.ts:7:17 - error TS2724: 'React' has no exported member named 'ForwardedRef'. Did you mean 'forwardRef'?

7     ref?: React.ForwardedRef<ChartJSOrUndefined<"bar", TData, TLabel>> | undefined;
                  ~~~~~~~~~~~~

node_modules/react-chartjs-2/dist/typedCharts.d.ts:10:17 - error TS2724: 'React' has no exported member named 'ForwardedRef'. Did you mean 'forwardRef'?

10     ref?: React.ForwardedRef<ChartJSOrUndefined<"radar", TData, TLabel>> | undefined;
                   ~~~~~~~~~~~~

node_modules/react-chartjs-2/dist/typedCharts.d.ts:13:17 - error TS2724: 'React' has no exported member named 'ForwardedRef'. Did you mean 'forwardRef'?

13     ref?: React.ForwardedRef<ChartJSOrUndefined<"doughnut", TData, TLabel>> | undefined;
                   ~~~~~~~~~~~~

node_modules/react-chartjs-2/dist/typedCharts.d.ts:16:17 - error TS2724: 'React' has no exported member named 'ForwardedRef'. Did you mean 'forwardRef'?

16     ref?: React.ForwardedRef<ChartJSOrUndefined<"polarArea", TData, TLabel>> | undefined;
                   ~~~~~~~~~~~~

node_modules/react-chartjs-2/dist/typedCharts.d.ts:19:17 - error TS2724: 'React' has no exported member named 'ForwardedRef'. Did you mean 'forwardRef'?

19     ref?: React.ForwardedRef<ChartJSOrUndefined<"bubble", TData, TLabel>> | undefined;
                   ~~~~~~~~~~~~

node_modules/react-chartjs-2/dist/typedCharts.d.ts:22:17 - error TS2724: 'React' has no exported member named 'ForwardedRef'. Did you mean 'forwardRef'?

22     ref?: React.ForwardedRef<ChartJSOrUndefined<"pie", TData, TLabel>> | undefined;
                   ~~~~~~~~~~~~

node_modules/react-chartjs-2/dist/typedCharts.d.ts:25:17 - error TS2724: 'React' has no exported member named 'ForwardedRef'. Did you mean 'forwardRef'?

25     ref?: React.ForwardedRef<ChartJSOrUndefined<"scatter", TData, TLabel>> | undefined;
                   ~~~~~~~~~~~~

node_modules/react-chartjs-2/dist/types.d.ts:1:37 - error TS2724: '"react"' has no exported member named 'ForwardedRef'. Did you mean 'forwardRef'?

1 import type { CanvasHTMLAttributes, ForwardedRef, ReactNode, MouseEvent } from 'react';
                                      ~~~~~~~~~~~~

  node_modules/@types/react/index.d.ts:801:14
    801     function forwardRef<T, P = {}>(render: ForwardRefRenderFunction<T, P>): ForwardRefExoticComponent<PropsWithoutRef<P> & RefAttributes<T>>;
                     ~~~~~~~~~~
    'forwardRef' is declared here.


Found 10 errors.
```

--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
     "commit": "cz",
     "release": "standard-version",
     "cleanPublish": "yarn test && clean-publish",
-    "enableGitHooks": "simple-git-hooks"
+    "updateGitHooks": "simple-git-hooks"
   },
   "dependencies": {},
   "peerDependencies": {

--- a/src/chart.tsx
+++ b/src/chart.tsx
@@ -1,9 +1,8 @@
 import React, { useEffect, useRef, forwardRef } from 'react';
-import type { ForwardedRef } from 'react';
 import { Chart as ChartJS } from 'chart.js';
 import type { ChartType, DefaultDataPoint } from 'chart.js';
 
-import type { ChartProps, TypedChartComponent } from './types';
+import type { ForwardedRef, ChartProps, TypedChartComponent } from './types';
 import {
   reforwardRef,
   cloneData,

--- a/src/types.ts
+++ b/src/types.ts
@@ -1,4 +1,4 @@
-import type { CanvasHTMLAttributes, ForwardedRef, ReactNode } from 'react';
+import type { CanvasHTMLAttributes, MutableRefObject, ReactNode } from 'react';
 import type {
   Chart,
   ChartType,
@@ -7,6 +7,11 @@ import type {
   DefaultDataPoint,
   Plugin,
 } from 'chart.js';
+
+export type ForwardedRef<T> =
+  | ((instance: T | null) => void)
+  | MutableRefObject<T | null>
+  | null;
 
 export interface ChartProps<
   TType extends ChartType = ChartType,

--- a/src/utils.ts
+++ b/src/utils.ts
@@ -1,4 +1,4 @@
-import type { ForwardedRef, MouseEvent } from 'react';
+import type { MouseEvent } from 'react';
 import type {
   ChartType,
   ChartData,
@@ -7,6 +7,8 @@ import type {
   ChartOptions,
   Chart,
 } from 'chart.js';
+
+import type { ForwardedRef } from './types';
 
 const defaultDatasetIdKey = 'label';
 

